src/dynamics/ads_player/ads_player.js

```diff
@@ -381,34 +364,61 @@ Scoped.define("module:Ads.Dynamics.Player", [
                     })
                 },
 
-                setEndCardBackground: function(width, height) {
+                captureAdEndCardBackground: function(width, height) {
                     const ad = this.get("ad");
                     if (ad) {
-                        this.updateEndCardImage(ad, width, height);
+                        this.generateEndCardImage(ad, width, height);
                     }
                 },
 
-                updateEndCardImage: function(ad, width, height) {
+                setEndCardBackground: function() {
+                    const width = this.getAdWidth();
+                    const height = this.getAdHeight();
+                    if (width && height) {
+                        if (this.adsManager && typeof this.adsManager.resize === "function") {
+                            this.adsManager.resize(
+                                width,
+                                height,
+                                google.ima.ViewMode.NORMAL
+                            );
+                        }
```

Comment:
Hi @EdmundsBankai, this part isn't related to setting end card background, it's something we need to do whenever the player is resized to ensure the IMA SDK has the correct sizing, so I don't think it should be wrapped inside a function called `setEndCardBackground`.

Reply:
yes, i didn't notice that, will update this